include/SFML/Network/Socket.hpp

```diff
@@ -31,12 +31,55 @@
 
 #include <SFML/Network/SocketHandle.hpp>
 
+#include <algorithm>
+#include <array>
+#include <cstdint>
+#include <cstring>
 #include <vector>
 
 
 namespace sf
 {
 class SocketSelector;
+////////////////////////////////////////////////////////////
+/// \brief Endian test
+///
+////////////////////////////////////////////////////////////
+inline bool isBigEndian()
+{
+    int one{1};
```

Comment:
```suggestion
    int one = 1;
```
We don't use braces for initialization in this context.